mhkit/wave/io/ndbc.py

```diff
@@ -415,7 +432,13 @@ def request_data(parameter, filenames, proxy=None, clear_cache=False, to_pandas=
                 # Cache the data after processing it if it exists
                 if year in ndbc_data[buoy_id]:
                     handle_caching(
-                        hash_params, cache_dir, data=ndbc_data[buoy_id][year]
+                        hash_params,
+                        cache_dir,
+                        cache_content={
+                            "data": ndbc_data[buoy_id][year],
+                            "metadata": None,
+                            "write_json": None,
+                        },
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args